tests/projects/project1/packages/hello-agent/Cargo.toml

```
@@ -1,20 +1,19 @@
 [package]
-name = "hello-agent"
+name = "pkg-b"
```

Maybe this file needs to be deleted?
```suggestion
name = "hello-agent"
```

---

Errant change.